docs/io.cozy.todos.md

```diff
@@ -0,0 +1,123 @@
+[Table of contents](README.md#table-of-contents)
+
+# Cozy Todos doctype
+
+## `io.cozy.todos.list`
+
+This doctype represents a list of todos:
+
+- `title`: {string} - The list title
+- `description`: {string} - The list description
+- `limitDate`: {date} - The limit date to complete all the items.
+
+### Relationships
+
+- `items`: {object} - The list items
+
+
+### Example
+
+```json
+{
+  "_id": "2165d9a310deadbeeffc08d54c45102",
+  "title": "Create a loan file",
+  "description": "Loan money to buy a house.",
+  "limitDate": "2020-02-01T23:59:59Z",
+  "relationships": {
+    "items": {
+      "data": [
+        {
+          "_id": "8bffbc24e67716b63e08c71bd0059fea",
+          "_type": "io.cozy.todos.item"
+        },
+        {
+          "_id": "68a74e03722f02b1a6fc6a4508015de0",
+          "_type": "io.cozy.todos.item"
+        }
+      ]
+    }
+  }
+}
+```
+
+
+## `io.cozy.todos.item`
+
+This doctype represents a todo item:
+
+- `label`: {string} - The label of the todo item, e.g. "Do stuff".
+- `done`: {bool} - Whether the item is done or not.
+- `limitDate`: {date} - The limit date to complete this item.
+- `completionDate`: {date} - The actual completion date of the item.
+
+### Relationships
+
+- `files`: {object} - Files linked to the todo item.
+- `contacts`: {object} - Contacts linked to the todo item.
+- `rules`: {object} - The rules used to match others relationships. See below or in the `rules` [doctype](https://docs.cozy.io/en/cozy-doctypes/docs/io.cozy.rules) for more details.
```

Comment:
```suggestion
- `rules`: {object} - The rules used to match other relationships. See below or in the `rules` [doctype](https://docs.cozy.io/en/cozy-doctypes/docs/io.cozy.rules) for more details.
```

Reply:
" The rules used to match other relationships." I have trouble understanding what this means when reading. I know that there is more informatin below but still, I think it could be clarified.